sim/mcuboot-sys/build.rs

```diff
@@ -83,6 +83,10 @@ fn main() {
     conf.debug(true);
     conf.flag("-Wall");
     conf.flag("-Werror");
+    /// FIXME: travis-ci still uses gcc 4.8.4 which defaults to std=gnu90.
```

Comment:
These comments need to be with just two slashes "//". "///" are doc comments that go only before certain declarations. This generates a warning on the nightly compiler.

Reply:
Fixed.